packages/test-commerce-sdk-react/app/pages/use-shopper-products.tsx

```diff
@@ -13,7 +13,8 @@ const ids = '25502228M,25503045M'
 
 const UseShopperProducts = () => {
     const {isLoading, error, data: result} = useProducts({
-        ids
+        // @ts-ignore
+        foo: ids
```

Comment:
I think we either create a page to render error for demo instead of doing it here. This page should demonstrate the working useProducts